protobuf/definition/execution.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package definition;
+option go_package = "github.com/mesg-foundation/core/protobuf/definition";
+
+// Statys represents a status of execution.
+enum Status {
+  Unknown = 0;
+  Created = 0x01;
+  InProgress = 0x02;
+  Completed = 0x04;
+  Failed = 0x08;
```

Comment:
why hexa?

Reply:
It's common for flags that you can perform `or`|`and` op. It's easier to see when you have more flags (0x01..0x10) rather then (1..16) but 1, 2, 4, 8, 16 are perfectly fine.

Reply:
bitmask makes total sense for the filter but doesn't make much sense in the definition itself.
It's kind of confusing if we have it in the execution, it means that we could have 2 or more status at the same time. I'm ok to keep it but at least make it clear with comments

Reply:
@krhubert have you tried to use it with a client?
I'm pretty sure gRPC will not accept enum mask in a parameter of type `definition.Status`.
See line 31 of file protobuf/api/api.proto
https://github.com/mesg-foundation/engine/pull/1006/files#diff-759dbd0083a04481d938c7f1f0b9e403R31